Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+apertium-streamparser = "*"
+chardet = "*"
+requests = "*"
+tornado = "==4.2.1"
+commentjson = "*"
+
+[dev-packages]
+coverage = "*"
+coveralls = "*"
+flake8-commas = "*"
+flake8-quotes = "*"
+flake8 = "==3.5.0"
```

Comment:
Is it possible for us to allow patch upgrades to this package? i.e. something like `^3.5.0`?

Reply:
I think so, some style issues would have to be resolved though.

Reply:
Even on patch versions? If so, I guess we can just create an issue to update it.

Reply:
Yep.

Reply:
Or just ignore the style conventions mentioned.